sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -315,9 +310,9 @@ protected ColumnVector buildAndPutOnDevice(int builderIndex) {
       return gcv;
     }
 
-    public void copyColumnar(ColumnVector cv, int colNum, boolean nullable, int rows) {
+    public void copyColumnar(ColumnVector cv, int colNum, boolean ignored, int rows) {
```

Comment:
Because this does not override anything should we delete ignored too? Or is there a reason to keep it?

Reply:
It implements GpuColumnarBatchBuilderBase#copyColumnar, and this parameter is used in the other implementation GpuColumnarBatchBuilder